sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -119,6 +145,21 @@ def __init__(self, xcvr_eeprom):
         self.vdm = CmisVdmApi(xcvr_eeprom) if not self.is_flat_memory() else None
         self.cdb = CmisCdbApi(xcvr_eeprom) if not self.is_flat_memory() else None
 
+    def clear_cache(self, method_name=None):
```

Comment:
@jianyuewu seems not used. let the cache clear automatically when Xcvrd restart.

Reply:
UT need this clear_cache😊 Otherwise, next test of same function with different output will fail. Though in offical code, cache is in CmisApi, when xcvrd process restart or recreate CmisApi object, old instance and cache will be deleted.

Reply:
@jianyuewu you can still move this to test code

Reply:
Agree, I can put it in test code, will move it.